docs/quickstart/apollo_2_0_hardware_system_installation_guide_v1.md

```diff
@@ -1,4 +1,4 @@
-# Apollo 2.0 Hardware and System Installation Guide
+# Apollo 2.5 Hardware and System Installation Guide
```

Comment:
This title is this file is "apollo_2_0_hardware_system_installation_guide_v1.md", which doesn't match the modified content.